service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1943,4 +1870,11 @@ public Set<String> getAllAuthorities(String email) {
     public void updateEmployeesAuthorities(UserEmployeeAuthorityDto dto, String email) {
         userRemoteClient.updateEmployeesAuthorities(dto, email);
     }
+
+    @Override
+    public List<LocationSummaryDto> getSummaryLocation() {
+        return regionRepository.findAll().stream()
+            .map(i -> modelMapper.map(i, LocationSummaryDto.class))
```

Comment:
please choose the better name for the local variables

Reply:
getSummaryLocation -> getLocationSummary